Jenkinsfile

```diff
@@ -20,134 +20,170 @@ pipeline {
     REPO_DIR="$GOPATH/src/github.com/intel/${REPO_NAME}"
   }
   stages {
-    stage("Get requirements") {
-      parallel {
-        stage("go") {
-          steps {
-            sh "curl -O https://dl.google.com/go/${GO_TAR}"
-            sh "tar -xvf $GO_TAR"
-            sh "sudo mv go $GOROOT"
-            sh "mkdir -p $GOPATH/src/github.com/intel"
-            sh "cp -rf ${env.WORKSPACE} $REPO_DIR"
-            dir(path: "$REPO_DIR") {
-              sh "go get -v golang.org/x/lint/golint"
-              sh "go get -v golang.org/x/tools/go/analysis/passes/shadow/cmd/shadow"
-              sh "go get -v github.com/fzipp/gocyclo"
-            }
-          }
-        }
-        stage("buildah") {
-          steps {
-            sh "sudo apt-get update"
-            sh "sudo apt-get -y install e2fslibs-dev libfuse-dev libgpgme11-dev libdevmapper-dev libglib2.0-dev libprotobuf-dev"
-            sh "mkdir -p ${GOPATH}/src/github.com/containers"
-            dir(path: "${GOPATH}/src/github.com/containers") {
-              sh "git clone --single-branch --depth 1 -b $BUILDAH_VERSION https://github.com/containers/buildah"
-            }
-            dir(path: "${GOPATH}/src/github.com/containers/buildah") {
-              sh 'make buildah TAGS=""'
-              sh "sudo cp buildah /usr/local/bin"
-              sh "sudo mkdir -p /etc/containers"
-              sh '''echo '[registries.search]' > registries.conf'''
-              sh '''echo 'registries = ["docker.io"]' >> registries.conf'''
-              sh "sudo mv registries.conf /etc/containers/registries.conf"
-              sh "sudo curl https://raw.githubusercontent.com/kubernetes-sigs/cri-o/$CRIO_VERSION/test/policy.json -o /etc/containers/policy.json"
-              sh "sudo curl -L https://github.com/opencontainers/runc/releases/download/$RUNC_VERSION/runc.amd64 -o /usr/bin/runc"
-              sh "sudo chmod +x /usr/bin/runc"
-            }
-          }
+    stage("Set env") {
+      when { changeRequest() }
+      steps {
+        script {
+          env.TAG = env.BUILD_TAG + '-rejected'
```

Comment:
does `BUILD_TAG` refer to the pull request ID? Maybe you meant `CHANGE_ID` here?

Reply:
done